config/routes.rb

```diff
@@ -12,7 +12,7 @@
 
   resources :users, only: [:index, :show, :destroy] do
     resources :posts, only: [:index, :new, :create, :show, :destroy] do 
-      resources :comments, only: [:new, :create, :destroy]
+      resources :comments, only: [:index, :new, :create, :destroy]
```

Comment:
- [ ] [OPTIONAL] Good job so far but I would like to recommend you to add /api to URL or api/v1 with the version number in order to identify it as an API-based URL because it's It's common practice 👍